scaffold/template/custom/public/index.html

```diff
@@ -3,9 +3,26 @@
 
     <head>
         <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0"
+        />
+        
         <meta name="theme-color" content="#000000">
         <title>React App</title>
+        <style>
+            html, body {
+                height: 100%;
+                margin: 0;
+                padding: 0;
+            }
+
+            #root {
+                height: 100%;
+                display: flex;
+                flex-direction: column;
+            }
+        </style>
```

Comment:
Did you test the impact of these changes on other screens/modules?

Reply:
im on it! takes time, i pushed the PR to have your feedback while im testing a few modules.

Reply:
@danielsousaio i tested a few modules and i didnt see any weird behavior regarding the new css